simplq/src/components/pages/Status/QueueStatus.jsx

```diff
@@ -83,8 +84,15 @@ function QueueStatus(props) {
     return <div />;
   };
 
+  const renderDetails = () => {
+    if (tokenStatusResponse.tokenStatus !== 'REMOVED') {
+      return <QueueDetails queueId={tokenStatusResponse.queueId} />;
+    }
+    return <div />;
+  };
```

Comment:
We can still show the queue details even if it was removed?

Reply:
Should we? My opinion is there's really no point. The queue name is still displayed at the top, but the other details need not be dislayed or updated, right?

Reply:
Ok. This brings the next point... we need to update QueueStatus in 10s interval... But this is a cross cutting concern that we need to handle at a single place. (Admin Page, Token Status Page and QueueDetails all need to keep refreshing, we don't want to repeat the logic). Pavan would work on this part.

Let's create an issue and move on.

Reply:
#170

Reply:
By saying richer, do you mean having more content on every page?

Reply:
Yes. But this is a product decision. Let's not do it for now.